Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -0,0 +1,334 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="BootstrapperExeTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
```

Comment:
There weren't any tests for the behaviour of the bootstrapper exe previously. These tests start to fill that test hole.